spring-cloud/spring-cloud-task/springcloudtaskbatch/pom.xml

```diff
@@ -43,6 +44,11 @@
             <groupId>org.springframework.cloud</groupId>
             <artifactId>spring-cloud-task-batch</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.h2database</groupId>
+            <artifactId>h2</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
The functionality analyzed in the article doesn't seem to MySql-specific, therefore I believe we're ok to use H2 for testing, so that we can leave the context test as an IntegrationTest, sounds good?